### PR TITLE
Update image library and gravity param

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

Updates Utopia-php/image version and changes the image crop gravity param names

## Test Plan

Existing Tests

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: app/controllers/api/storage.php [hunk 1]
```diff
@@ -242,7 +242,7 @@
     ->param('fileId', '', new UID(), 'File unique ID')
     ->param('width', 0, new Range(0, 4000), 'Resize preview image width, Pass an integer between 0 to 4000.', true)
     ->param('height', 0, new Range(0, 4000), 'Resize preview image height, Pass an integer between 0 to 4000.', true)
-    ->param('gravity', Image::GRAVITY_CENTER, new WhiteList([Image::GRAVITY_CENTER, Image::GRAVITY_NORTH, Image::GRAVITY_NORTHWEST, Image::GRAVITY_NORTHEAST, Image::GRAVITY_WEST, Image::GRAVITY_EAST, Image::GRAVITY_SOUTHWEST, Image::GRAVITY_SOUTH, Image::GRAVITY_SOUTHEAST]), 'Image crop gravity', true)
+    ->param('gravity', Image::GRAVITY_CENTER, new WhiteList([Image::GRAVITY_CENTER, Image::GRAVITY_TOP, Image::GRAVITY_TOP_LEFT, Image::GRAVITY_TOP_RIGHT, Image::GRAVITY_LEFT, Image::GRAVITY_RIGHT, Image::GRAVITY_BOTTOM_LEFT, Image::GRAVITY_BOTTOM, Image::GRAVITY_BOTTOM_RIGHT]), 'Image crop gravity', true)
```
Comment (eldadfux) [inline app/controllers/api/storage.php:245]: Just realized we must mention the different options in the description as well. Maybe we can add a static method to get all the types in the Image library? `Image::getGravityTypes():array`

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: app/controllers/api/storage.php (hunk 1 above)
Comment (lohanidamodar) [reply app/controllers/api/storage.php:245]: We can.

Comment (lohanidamodar) [reply app/controllers/api/storage.php]: I've updated, but needs release of `utopia-php/image` to be updated as well

Comment (eldadfux) [reply app/controllers/api/storage.php]: Great, I have released v0.5.0

**Pull request merged**